src/lib/cli/format.js

```diff
@@ -122,14 +122,27 @@ export function requoteArgs( args: Array<string> ): Array<string> {
 			return arg.replace( /^--(.*)=(.*)$/, '--$1="$2"' );
 		}
 
-		if ( arg.includes( ' ' ) ) {
+		if ( arg.includes( ' ' ) && ! isJsonObject( arg ) ) {
 			return `"${ arg }"`;
 		}
 
 		return arg;
 	} );
 }
 
+export function isJsonObject( str: string ): boolean {
+	return typeof str === 'string' && str.startsWith( '{' ) && isJson( str );
```

Comment:
An edge case but, `JSON.parse( '       {}       ' )` returns `{}`

Reply:
@abdullah-kasim Good catch. I've added some trimming before checking the `startsWith`.  This way if it's a valid JSON Object, `isJson` will return true.

Updated in https://github.com/Automattic/vip-cli/pull/1237/commits/cd4f1161024a3b2098bdcf414634dbc95dac9dc6